climada/engine/test/test_impact.py

```diff
@@ -69,6 +69,119 @@ def test_from_eih_pass(self):
             np.stack([exp.gdf.latitude.values, exp.gdf.longitude.values], axis=1)
             )
 
+    def test_concat_pass(self):
```

Comment:
I think it is better to separate these test into sub methods. Otherwise, if something breaks, it is hard to see what from the report.